client/src/components/light/lights.tsx

```diff
@@ -18,8 +18,52 @@ along with Home Lights.  If not, see <http://www.gnu.org/licenses/>.
 */
 
```

Comment:
Can we rename this component to LightList?

Reply:
It does more than show a list, since it also has the create light button in it. This file is directly analogous to the zone/zones.tsx file, and I followed the naming scheme there.